libs/sdk-ui-dashboard/src/hooks/useInsightWidgetDataView.ts

```diff
@@ -1,42 +1,56 @@
-// (C) 2020-2021 GoodData Corporation
-import { useMemo } from "react";
-import stringify from "json-stable-stringify";
+// (C) 2022 GoodData Corporation
 import { IInsightWidget } from "@gooddata/sdk-backend-spi";
-import { insightSetFilters } from "@gooddata/sdk-model";
 import {
+    DataViewFacade,
```

Comment:
I think this file should live in libs/sdk-ui-dashboard/src/model/react folder, there are already other hooks there. What do you think?